odoo/models.py

```diff
@@ -1697,6 +1700,27 @@ def _compute_display_name(self):
             for record in self:
                 record.display_name = f"{record._name},{record.id}"
 
+    @api.model
```

Comment:
This decorator only makes sense for public methods
```suggestion
```

Reply:
I know it is not doing anything, but for me it is documentation saying "we do not use ids of self" in this method. It is a kind of classmethod, but we need an instance to get the environment.
Isn't it a valid usage?